src/Android/Xamarin.Android/Samples/Augmented Reality/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.cs

```diff
@@ -0,0 +1,178 @@
+
+                _helpLabel.Text = "Tap the dots to place scene.";
+            }
+        }
+
+        private void ArSceneView_GeoViewTapped(object sender, Esri.ArcGISRuntime.UI.Controls.GeoViewInputEventArgs e)
+        {
+            // Get the tapped screen point.
+            PointF screenPoint = e.Position;
+
+            if (_arSceneView.SetInitialTransformation(screenPoint))
+            {
+                DisplayScene();
+                _helpLabel.Visibility = ViewStates.Gone;
+            }
+            else
+            {
+                Toast.MakeText(this, "ARCore doesn't recognize that point as a plane.", ToastLength.Short);
+            }
+        }
+
+        private async void DisplayScene()
+        {
+            try
+            {
+                if (_tabletopScene == null)
+                {
+                    // Get the downloaded mobile scene package.
+                    MobileScenePackage package = await MobileScenePackage.OpenAsync(DataManager.GetDataFolder("7dd2f97bb007466ea939160d0de96a9d", "philadelphia.mspk"));
+
+                    // Load the package.
+                    await package.LoadAsync();
+
+                    // Get the first scene.
+                    _tabletopScene = package.Scenes.First();
+
+                    // Hide the base surface.
+                    _tabletopScene.BaseSurface.Opacity = 0;
+
+                    // Enable subsurface navigation. This allows you to look at the scene from below.
+                    _tabletopScene.BaseSurface.NavigationConstraint = NavigationConstraint.None;
+                }
+
+                // Display the scene.
+                _arSceneView.Scene = _tabletopScene;
+
+                // Create a camera at the bottom and center of the scene.
+                //    This camera is the point at which the scene is pinned to the real-world surface.
+                var originCamera = new Esri.ArcGISRuntime.Mapping.Camera(39.95787000283599,
+                                                                        -75.16996728256345,
+                                                                        8.813445091247559,
+                                                                        0, 90, 0);
+
+                // Set the origin camera.
+                _arSceneView.OriginCamera = originCamera;
+
+                // The width of the scene content is about 800 meters.
+                double geographicContentWidth = 800;
+
+                // The desired physical width of the scene is 1 meter.
+                double tableContainerWidth = 1;
+
+                // Set the translation facotr based on the scene content width and desired physical size.
+                _arSceneView.TranslationFactor = geographicContentWidth / tableContainerWidth;
+            }
+            catch (System.Exception ex)
+            {
+                System.Diagnostics.Debug.WriteLine(ex);
+                ShowMessage("Failed to load scene.", "Error");
```

Comment:
You can replace the ShowMessage method with
`new AlertDialog.Builder(this).SetMessage("Failed to load scene.").SetTitle("Error").Show();`